packages/manager/modules/iam/src/components/createResourceGroup/createResourceGroup.styles.scss

```diff
@@ -0,0 +1,3 @@
+.iam-create_resource_group {
+  max-width: 40rem;
```

Comment:
**nitpick:** Use bootstrap utility instead.

Reply:
Can you tell me witch one ?

Reply:
Okay, I see we have only `mw-100` with respect to `max-width`. 
But can you use row and have a col of fixed width always?